activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -638,7 +638,7 @@ def attempt_to_checkout_all_existing_connections(raise_on_acquisition_timeout =
           raise
         end
       rescue Exception # if something else went wrong
-        # this can't be a "naked" rescue, because we have should return conns
+        # this can't be a "naked" rescue, because we should have return conns
```

Comment:
This isn't public documentation but internal comments, I'd prefer to keep the git history here — can you change it back? :grin: